ci/trigger_ci.sh

```diff
@@ -106,29 +106,41 @@ then
     mark_failure "Error while deploying the service in CICO"
 fi
 
-echo "Cluster is set lets go for tests"
+echo "Build jenkins slave to include current code base"
+ssh $sshoptserr $openshift_1_node_ip "cd /opt/ccp-openshift/Dockerfiles/ccp-openshift-slave && docker build -t $nfs_node:5000/pipeline-images/ccp-openshift-slave:latest . && docker push $nfs_node:5000/pipeline-images/ccp-openshift-slave:latest"
```

Comment:
The docker build command should be invoked from root dir of git repo to make sure the complete source code is added in container.
In Dockerfile, we have
```
ADD ./ /opt/ccp-openshift/
```

meaning,
```
cd /opt/ccp-openshift && docker build -t $nfs_node:5000/pipeline-images/ccp-openshift-slave:latest -f Dockerfiles/ccp-openshift-slave/Dockerfile . && docker push $nfs_node:5000/pipeline-images/ccp-openshift-slave:latest
```

The buildcontext here should be root dir, so that when we say `ADD ./ /opt/ccp-openshift/` complete source code is added.

```
# pwd
/root/container-pipeline-service/Dockerfiles/ccp-openshift-slave
#docker build -t pipeline-images/ccp-openshift-slave:latest .
# docker run -t --entrypoint ls pipeline-images/ccp-openshift-slave:latest /opt/ccp-openshift/
Dockerfile  README.md  cccp.yml
```